pkgs/by-name/ca/catppuccin-papirus-folders/package.nix

```diff
@@ -31,27 +33,43 @@ let
     "mocha"
   ];
   pname = "catppuccin-papirus-folders";
+
+  # Note(sewer56):
+  # Fetch the papirus-folders script from upstream
+  # Per instructions in the papirus-folders project.
+  # We fetch specific latest tested commit to ensure reproducibility.
```

Comment:
This last sentence kinda goes without saying in nixpkgs; all external inputs are always pinned. No need for explanation.


```suggestion
```